src/plugins/navigation/public/top_nav_menu/_index.scss

```diff
@@ -9,7 +9,7 @@
 }
 
 .osdTopNavMenuGroupedActions {
-  background-color: $euiColorEmptyShade;
+  background-color: $euiPageBackgroundColor;
```

Comment:
nit: think you could just make this transparent to get buttons to be consistent with how buttons normally behave in case their styling changes

Reply:
Thanks. Will do that in a subsequent PR.